api.py

```diff
@@ -204,3 +215,15 @@ def resetDefaultPrograms():
             with open("data/defaults/programs/" + filename) as p:
                 q = p.read()
                 programs.insert(json.loads(q))
+
+## Reset
+def reset():
+    pi = pigpio.pi('localhost')
+    #simulating FALLING EDGE
```

Comment:
Maybe put more detail on the fact we are actually triggering a button pression, handled by the service you implemented in another place?